ops/internal/deployer/versions.json

```diff
@@ -0,0 +1,8 @@
+{
+  "op-contracts/v1.6.0": "op-deployer/v0.0.14",
+  "op-contracts/v1.7.0-beta.1+l2-contracts": "op-deployer/v0.0.14",
+  "op-contracts/v1.8.0": "op-deployer/v0.0.14",
+  "op-contracts/v2.0.0": "op-deployer/v0.2.2",
+  "op-contracts/v3.0.0": "op-deployer/v0.3.1",
+  "op-contracts/v4.0.0": "op-deployer/v0.4.0-rc.2"
+}
```

Comment:
The `-rc.x` suffix for `op-contracts` is stripped before using this map so, e.g. any `op-contracts/v2.0.0*` will map to `op-deployer/v0.2.2`